app/interactors/show_group.rb

```diff
@@ -1,3 +1,9 @@
-class ShowGroup
-  include Interactor
+class ShowGroup < StandardInteraction
+  def validate_input
+    context.fail!(errors: "invalid input") unless context.id
+  end
+
+  def execute
+    context.group = Group.find(context.id)
+  end
```

Comment:
@Enriikke so here I'm not validating output. If there isn't a `Group` that matches the `id`, a `RecordNotFound` error is thrown. Back to the discussion above, I'm not sure if this is preferable to rescuing the error and add a custom error to `InteractorContext.errors`